src/Model/Generated/Endpoint/BunqMeTabResultInquiry.php

```diff
@@ -50,6 +71,10 @@ public function isAllFieldNull()
             return false;
         }
 
+        if (!is_null($this->bunqMeTabId)) {
+            return false;
+        }
+
```

Comment:
You can make this one if/elseif/else.

Reply:
Thats hard to generate as it requires some good logic to know when to stop the else if chain and do else. This was a fast and easy solution. Will keep this as improvement.

Reply:
Ah right,.. it is generated code. 😅  Would be better to review the generator and have proper tests for it than reviewing the generated code itself imho.

Reply:
The generator is ofc also review 😝 but its not open source 😉